http/src/client/mod.rs

```diff
@@ -1478,6 +1478,16 @@ impl Client {
         ExecuteWebhook::new(self, webhook_id, token)
     }
 
+    /// Get a webhook message by [`WebhookId`], Token, and [`MessageId`].
```

Comment:
These need to be linked

Reply:
They're linked by the import

Reply:
Those aren't, the ones in the request file are. whoops